osvimdriver/openstack/heat/driver.py

```diff
@@ -18,7 +23,7 @@ def __init__(self, session):
     def __get_heat_client(self):
         return self.__heat_client
 
-    def create_stack(self, stack_name, heat_template, input_properties=None, files=None):
+    def create_stack(self, stack_name, heat_template, input_properties=None,  files=None,driver_request_id=None ):
```

Comment:
create_stack function should not pass driver_request_id, as this function generates one and need to be allowed only in the response logs. request logs driver_request_id can be removed.

Reply:
I have done changes.